src/components/EventDistances/index.js

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+
+export default function EventDistanceButtons({ onSelect }) {
+  const [selectedType, setSelectedType] = useState("short");
+
+  function handleTypeChange(event) {
+    setSelectedType(event.target.value);
+    onSelect(event.target.value);
+  }
+
+  return (
+    <form>
+      <label>
+        Short Distance Triathlon
+        <button
+          type="button"
+          className={selectedType === "short" ? "selected" : ""}
+          onClick={() => handleTypeChange({ target: { value: "short" } })}
+        >
+          {selectedType === "short" ? "Selected" : "Select"}
+        </button>
+      </label>
+      <br />
+      <br />
+      <label>
+        Middle Distance Triathlon
+        <button
```

Comment:
Maybe use radio buttons instead of buttons, since this is a more appropriate UI element for selecting a single option from a set of options. This would also simplify the code, since the selected value would be automatically managed by the form.